docs/api-docs/in-depth-guides/rail-integration-guide.mdx

```diff
@@ -1,100 +1,154 @@
+description: 'This guide provides a comprehensive, step-by-step approach for integrating North American Class-1 rail container tracking data into your systems. Whether you are a shipper or a logistics service provider, this guide will help you track all your rail containers via a single API.'
 mode: "wide"
 
 ---
 
 
-## Terminal40 Rail Integrations Overview
+This is a technical article about rail data within Terminal49's API and DataSync.
 
-Terminal49 container tracking platform integrates with all major North American Class-1 railroads, providing comprehensive visibility into rail container movements. Our platform supports the following carriers:
+For a broader overview, including the reasons why you'd want rail visibility and how to use it in the Terminal49 dashboard,
+ [read our announcement post](https://www.terminal49.com/blog/launching-north-american-intermodal-rail-visibility-on-terminal49/).
+
+## Table of Contents
+
+- [Supported Rail Carriers](#supported-rail-carriers)
+- [Supported Rail Milestones and Data Attributes](#supported-rail-milestones-and-data-attributes)
+  - [Rail Milestones (aka rail events)](#rail-milestones-aka-rail-events)
+  - [Webhook Notifications](#webhook-notifications)
+  - [Rail Container Attributes](#rail-container-attributes)
+- [Integration Methods](#integration-methods)
+  - [Integration via API](#a-integration-via-api)
+  - [Integration via DataSync](#b-integration-via-datasync)
+
+## Supported Rail Carriers
+
+Terminal49 container tracking platform integrates with all North American Class-1 railroads that handle container shipping, providing comprehensive visibility into your rail container movements.
 
-### Supported Rail Carriers
 - BNSF Railway
 - Canadian National Railway (CN)
 - Canadian Pacific Railway (CP)
 - CSX Transportation
-- Kansas City Southern Railway (KCS)
 - Norfolk Southern Railway (NS)
 - Union Pacific Railroad (UP)
 
 By integrating with these carriers, Terminal49 ensures that you have real-time access to critical tracking data, enabling better decision-making and operational efficiency.
 
-### Supported Rail Milestones and Data Attributes
+## Supported Rail Milestones and Data Attributes
+
+Terminal49 seamlessly tracks your containers as they go from container ship, to ocean terminal, to rail carrier.
+
+We provide a set of milestones/events that let you track the status of your containers as they move through the rail system.  You can be notified by webhook whenever these events occur.
+
+We also provide a set of attributes that let you know the current status of your container at any given time, as well as useful information such as ETA, pickup facility, and availability information.
 
-Terminal49 tracks various milestones and data attributes that are essential for effective rail container management. The key events and attributes include:
+### Rail Milestones (aka rail events)
```

Comment:
Given this is the API docs, I think should be specific. I'd suggest
> Rail Specific Transport Events

Reply:
Updated ✅